go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -153,6 +180,17 @@ func DemystifyPending(status v1.PodStatus) (pluginsCore.PhaseInfo, error) {
 							finalMessage := fmt.Sprintf("%s|%s", c.Message, containerStatus.State.Waiting.Message)
 							switch reason {
 							case "ErrImagePull", "ContainerCreating", "PodInitializing":
```

Comment:
Anand mentioned there are a couple more errors we should add here... 
@anandswaminathan can you remind me?

Reply:
Yes. We can follow up, should be easy to add